Dockerfile

```diff
@@ -127,8 +127,8 @@ ENV CASSANDRA_CLUSTER=false \
 # Configure search nodes. Empty string falls back to search being done on the nodes themselves.
 ENV PEGA_SEARCH_URL=
 
-# Configure hazelcast. By default, hazelcast runs in embedded mode.
-ENV HZ_CLIENT_MODE=false \
+# Configure hazelcast.
+ENV HZ_CLIENT_MODE=true \
```

Comment:
Are these changes only applicable for pega platform version greater than 8.6.
This is the base image which will be used for creating pega fully baked image. So even when 8.5.x patch releases which are going to be released in future will have hazelcast client-server mode on by default?
Will it cause any issue in 8.5.x future images?